content/src/content/jcr_root/apps/core/wcm/sandbox/components/teaser/v1/teaser/_cq_dialog/.content.xml

```diff
@@ -22,46 +22,47 @@
                             sling:resourceType="granite/ui/components/coral/foundation/container"
                             margin="{Boolean}true">
                         <items jcr:primaryType="nt:unstructured">
-                            <withCTA
+                            <actionsEnabled
                                     jcr:primaryType="nt:unstructured"
                                     sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                    fieldDescription="Check to allow adding Call To Action (CTA) elements."
+                                    fieldDescription="Check to allow adding Call To Action elements."
                                     text="With Call-To-Action(s)"
-                                    name="./withCTA"
+                                    name="./actionsEnabled"
                                     value="{Boolean}true"
                                     uncheckedValue="{Boolean}false"
-                                    granite:hide="${cqDesign.disableCTA}">
-                            </withCTA>
-                            <ctas
+                                    granite:hide="${cqDesign.disabled}">
+                            </actionsEnabled>
+                            <actions
```

Comment:
There's some strange behavior with the validation of the link / multifield. 

If I:
* Allow actions via the "with Call-To-Actions" checkbox (I'm informed that the first link of the first action is required).
* Disable actions again.
-> Result: It's not possible to confirm the dialog any longer
-> Expectation: Can always confirm the dialog if there is no link and actions are disabled